tests/test_dia_calculation_plugins.py

```diff
@@ -457,6 +458,56 @@ def testCalculate(self):
         self.assertAlmostEqual(diaObject["rPSFluxLinearIntercept"], -1.)
 
 
+class TestStetsonJDiaPsFlux(unittest.TestCase):
+
+    def testCalculate(self):
+        """Test the stetsonJ statistic.
+        """
+        n_sources = 10
+        diaObject = dict(uPSFluxMean=0)
+        fluxes = np.linspace(-1, 1, n_sources)
+        errors = np.ones(n_sources)
+        times = np.linspace(0, 1, n_sources)
+        diaSources = pd.DataFrame(data={"psFlux": fluxes,
+                                        "psFluxErr": errors,
+                                        "midPointTai": times})
+
+        plug = StetsonJDiaPsFlux(LinearFitDiaPsFluxConfig(),
+                                 "ap_LinearFit",
```

Comment:
Should this be changed to Stetson things?

Reply:
No, the StetsonJ statistic is the only thing stored.